homework/anastasiadom/nastia_hw5.py

```diff
@@ -0,0 +1,10 @@
+import re
+import sys
+
+def StrPolindrom(s):
+	s1=s.lower()
```

Comment:
you do not need 3 variables make it in one line

Reply:
done